.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: circleci/node:9.8.0-browsers
+      - image: circleci/node:11
```

Comment:
Odd versions of Node are unsupported development builds. Never ever use an odd version of Node unless you're working on Node.js itself or you want to test your package against a new feature before it's released. Node.js 10.x or Node.js 12.x is what should be used.

Reply:
Node.js 11 isn't even supported currently.

Reply:
I bumped it up to 12